### ISSUE TITLE
reghack was obsolted by OpenWrt r45252

### ISSUE THREAD

**Issue by dangowrt** [battlemesh/wibed-packages#1]
it should no longer be needed. Anyone?

**Comment by emmdim** [comment]
Wibed is still running r42454 i think. So we need to upgrade it before removing.
Pau, do you think you can upgrade?

**Comment by dangowrt** [comment]
I tried putting the 4 wibed/battlemesh specific patches on top of todays OpenWrt trunk, removed all the regulatory hacking stuff which should no longer be needed (and if so, can be re-introduced part-by-part)

https://github.com/dangowrt/wibed-cc

**Comment by ghost** [comment]
In branch wbmv8 is removed:
https://github.com/battlemesh/wibed-packages/commit/5d50162688da1379120e90028a4d80f9ac2ec8c1#diff-73fe910fb7d551eee7554e161b5badabR3

**Comment by dangowrt** [comment]
Did that obscure bug making the choise of either DFS-FCC or DFS-ETSI affect transmit power on 2.4GHz affect us as well?
https://lists.openwrt.org/pipermail/openwrt-devel/2015-August/034829.html
Apart from that, I reckon we can close this ticket...?

**Comment by emmdim** [comment]
Current wibed master branch is still on barrier breaker stable. Once Chaos Calmer is out and we update the firmware we can close this. Is that ok with you?

**Comment by bittorf** [comment]
also i want to point out, that reghack was never needed, because we dont need DFS channels and we reduce the power anyway - do you agree?

**Comment by p4u** [comment]
Yes in the WBM case, but others who use Wibed to deploy their testbeds may want to use DFS channels and high power. As far as Wibed is designed to be used for research purposes it should allow the maximum number of possible scenarios (IMO).

cheers

On 12 agost de 2015 11:55:14 CEST, Bastian Bittorf notifications@github.com wrote:

> also i want to point out, that reghack was never needed, because we
> dont need DFS channels and we reduce the power anyway - do you agree?
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/battlemesh/wibed-packages/issues/1#issuecomment-130242998

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by zavalyshyn** [comment]
Even if we don't disable it, current version of reghack that is part of
wibed-packages doesn't work with the latest OpenWRT trunk. When selected,
reghack leaves the channels settings as they were by default.

**Comment by bittorf** [comment]
we are using internally a patch, which patches mac80211/ath9k on compile time:
https://github.com/bittorf/kalua/blob/master/openwrt-patches/reghack/900-regulatory-compliance_test.patch
